lib/ueberauth/strategy/line/oauth.ex

```diff
@@ -47,9 +47,10 @@ defmodule Ueberauth.Strategy.Line.OAuth do
   end
 
   def get_token!(params \\ [], opts \\ []) do
-    token = opts
-    |> client
-    |> OAuth2.Client.get_token!(params)
+    token =
+      opts
+      |> client
+      |> OAuth2.Client.get_token!(params)
 
     token
```

Comment:
I think we can omit `token` var here
```suggestion
      opts
      |> client
      |> OAuth2.Client.get_token!(params)
```